src/components/table/TablePagination.js

```diff
@@ -221,8 +222,10 @@ class TablePagination extends Component {
                     }}
                 >
                     {selectedWholePage ?
-                        'Select all ' + size + ' items' :
-                        'Select all on this page'
+                        counterpart.translate(
+                            'view.selectAll.caption'
+                        ) + size + ' items' :
```

Comment:
ok we will go with that. This is possible to have parameters in message e.g counterpart.translate('example.you_clicked_on', { what: this.props.name })

and in key map we need something like this: you_clicked_on: 'You clicked on %(what)s.'